source/docs/advanced/graphql/dataloaders-and-cache-invalidation.md

```diff
@@ -406,6 +406,37 @@ export default {
 };
 ```
 
+### `PerMagentoAdminRole`
+
+<blockquote class="feature--new">
+_Since version 2.1.0_
+</blockquote>
+
+The `PerMagentoAdminRole` implementation is a decorator that is specific to Magento 2 module. It will decorate the existing caching strategies so that DataLoader keys are different for admin users and customers. This is useful if a custom Magento API returns a different result for admins and normal customers.
```

Comment:
```suggestion
The `PerMagentoAdminRole` implementation is a decorator that is specific to Magento 2 integrations. It will decorate the existing caching strategies so that DataLoader keys are different for admin users and customers. This is useful if a custom Magento API returns a different result for admins and normal customers.
```

I wonder if it wouldn't make sense to start adding a tag system to features depending on the integrations where it is supported. Something like the `feature--new` with a visual tag-like system.